cat-launcher/src-tauri/src/analytics/helpers.rs

```diff
@@ -0,0 +1,29 @@
+use std::collections::HashMap;
+
+use tauri::{AppHandle, Manager, State};
+
+use crate::{
+    analytics::service::Analytics,
+    users::{repository::sqlite_users_repository::SqliteUsersRepository, service::get_or_create_user_id},
+};
+
+pub async fn track_event(
+    handle: &AppHandle,
```

Comment:
Instead of accepting the entire handle, accept only what's needed as parameters: Analytics, and UsersRepository.